elastic/kibana
# [Reporting] Allow reporting capabilities on gold with the new reporting roles

## Summary

Close https://github.com/elastic/kibana/issues/127020

After we required users to migrate to using the new Kibana driven reporting roles a bug surfaced that prevents usage of PDF or PNG reports.

## How to test

1. Start a fresh Kibana and ES
2. Upload a Gold license
3. Check that PDF and PNG reporting is available

## Release note

We fixed a bug that prevented Gold license users from accessing PDF and PNG reporting features when using Kibana roles.

## Checklist

Tried adding a simple test, but it is not currently supported use a `gold` license in a functional test (might be wrong). Happy to open a follow up issue for this. Currently only been manually tested.
